src/core/src/bootstrap/Constants.py

```diff
@@ -219,10 +219,21 @@ class StatusTruncationConfig(EnumBackport):
     MAX_INSTALLATION_RETRY_COUNT = 3
     MAX_IMDS_CONNECTION_RETRY_COUNT = 5
     MAX_ZYPPER_REPO_REFRESH_RETRY_COUNT = 5
-    MAX_BATCH_SIZE_FOR_PACKAGES = 6
-    NUMBER_OF_PACKAGES_IN_BATCH_COULD_TAKE_MAX_TIME_TO_INSTALL = 3
     MAX_COMPLETE_STATUS_FILES_TO_RETAIN = 10
 
+    # Batch Patching Parameters
+    MAX_BATCH_SIZE_FOR_PACKAGES = 300
+    MAX_PHASES_FOR_BATCH_PATCHING = 2
+
+    # Batch size decay factor is factor to which batch size is decreased in batch patching to install remaining packages in case there
+    # are some package install failures with original batch size.
+    BATCH_SIZE_DECAY_FACTOR = 10
+
+    # We need to keep some buffer time between calculation of batch size and starting batch patching because after calculating the batch size,
+    # there would be little time taken before the batch patching is started. The function is_package_install_time_available is called before installing a batch.
+    # If we do not keep buffer then is_package_install_time_available would return false.
+    BUFFER_TIME_FOR_BATCH_PATCHING_START_IN_MINUTES = 5
```

Comment:
Grouping config by proximity in the file is okay given that's the pattern in master. But everything in this file is being refactored to logically group related config, and you aligning with that model will avoid me having to refactor your code to merge.

See this Constants file from the last published iteration (of the long-delayed PR close) for concrete examples - https://github.com/Azure/LinuxPatchExtension/blob/173c52230f8c0f02f7fbf290006d4b67ff211928/src/core/src/bootstrap/Constants.py

So a class like PackageBatchConfig(EnumBackport) with variables shortened with the class context:
- PackageBatchConfig.MAX_BATCH_SIZE                    (for packages is implicit and understood)
- .DECAY_FACTOR
- .MAX_PHASES
- .BATCHING_START_BUFFER_IN_MINUTES                (name suggestion to shorten)

The comments are valuable and should be retained. They may be combined at the class level as a multi-line if it helps.

Also, the way to save time in doing this is to first define the class in Constants. Then do a string replace (not a symbol replace) on the original constant name with PackageBatchConfig.CONSTANT